test/benchee/unit/duration_test.exs

```diff
@@ -1,7 +1,7 @@
-defmodule Benchee.Unit.DurationTest do
+defmodule Benchee.Conversion.DurationTest do
   use ExUnit.Case
-  import Benchee.Unit.Duration
-  doctest Benchee.Unit.Duration
+  import Benchee.Conversion.Duration
+  doctest Benchee.Conversion.Duration
```

Comment:
great side effect of importing the modules: we don't have to change all the module name spaces in the tests. Robustness++ :)